src/routes/controllers/tags.controller.js

```diff
@@ -76,6 +72,34 @@ export async function viewTags(req, res) {
   }
 }
 
+/**
+ * @description find tag
+ * @param {object} req
+ * @param {object} res
+ * @returns {Object} tag and articles
+ */
+export async function findTag(req, res) {
```

Comment:
we should rename this `getTag` or `getArticleTag`

Reply:
done